{"uid":"a9b9b848005a3ae","name":"test_cursor_multi_statement","fullName":"tests.unit.V1.db.test_cursor#test_cursor_multi_statement","historyId":"31cb49a095de720cccc4c534065740ba","time":{"start":1730430884608,"stop":1730430884621,"duration":13},"description":"executemany with multiple parameter sets is not supported.","descriptionHtml":"<p>executemany with multiple parameter sets is not supported.</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","statusTrace":"self = <allure_pytest.listener.AllureListener object at 0x7efd677b94c0>\nfixturedef = <FixtureDef argname='python_query_data' scope='function' baseid='tests/unit'>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_fixture_post_finalizer(self, fixturedef):\n        yield\n        if hasattr(fixturedef, 'cached_result') and self._cache.get(fixturedef):\n            container_uuid = self._cache.pop(fixturedef)\n>           self.allure_logger.stop_group(container_uuid, stop=now())\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_pytest/listener.py:190: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/reporter.py:91: in stop_group\n    plugin_manager.hook.report_container(container=group)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:34: in report_container\n    self._report_item(container)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:25: in _report_item\n    with io.open(self._report_dir / filename, 'w', encoding='utf8') as json_file:\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_io.py:93: in open\n    return fake_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:105: in fake_open\n    return fake_file_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:136: in __call__\n    return self.call(*args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:244: in call\n    file_object = self._init_file_object(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:358: in _init_file_object\n    file_object = self.filesystem.create_file_internally(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:2551: in create_file_internally\n    self.raise_os_error(errno.ENOENT, parent_directory)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pyfakefs.fake_filesystem.FakeFilesystem object at 0x7efd657def40>\nerr_no = 2\nfilename = '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\nwinerror = None\n\n    def raise_os_error(\n        self,\n        err_no: int,\n        filename: Optional[AnyString] = None,\n        winerror: Optional[int] = None,\n    ) -> NoReturn:\n        \"\"\"Raises OSError.\n        The error message is constructed from the given error code and shall\n        start with the error string issued in the real system.\n        Note: this is not true under Windows if winerror is given - in this\n        case a localized message specific to winerror will be shown in the\n        real file system.\n    \n        Args:\n            err_no: A numeric error code from the C variable errno.\n            filename: The name of the affected file, if any.\n            winerror: Windows only - the specific Windows error code.\n        \"\"\"\n        message = os.strerror(err_no) + \" in the fake filesystem\"\n        if winerror is not None and sys.platform == \"win32\" and self.is_windows_fs:\n            raise OSError(err_no, message, filename, winerror)\n>       raise OSError(err_no, message, filename)\nE       FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:489: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"clear_cache","time":{"start":1730430884326,"stop":1730430884326,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"global_fake_fs","time":{"start":1730430884326,"stop":1730430884331,"duration":5},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"global_fake_fs::0","time":{"start":1730430884650,"stop":1730430884651,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests.unit.V1.db"},{"name":"suite","value":"test_cursor"},{"name":"host","value":"fv-az775-215"},{"name":"thread","value":"2087-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.unit.V1.db.test_cursor"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":47,"skipped":0,"passed":58,"unknown":0,"total":105},"items":[{"uid":"9de7f49d62b172a0","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/341//#testresult/9de7f49d62b172a0","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730344327810,"stop":1730344327823,"duration":13}},{"uid":"23121ac7d7e9ea93","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/344//#testresult/23121ac7d7e9ea93","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730344197042,"stop":1730344197055,"duration":13}},{"uid":"2a680663ba765593","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/340//#testresult/2a680663ba765593","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730257881161,"stop":1730257881174,"duration":13}},{"uid":"a815e309b11f6f1b","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/343//#testresult/a815e309b11f6f1b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730257739648,"stop":1730257739662,"duration":14}},{"uid":"512a2b01511d8182","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/339//#testresult/512a2b01511d8182","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730171510309,"stop":1730171510322,"duration":13}},{"uid":"2f80f32b97d98b2b","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/342//#testresult/2f80f32b97d98b2b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730171372251,"stop":1730171372264,"duration":13}},{"uid":"e7573e5866bc44e5","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/338//#testresult/e7573e5866bc44e5","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730085242460,"stop":1730085242474,"duration":14}},{"uid":"547aba3eed5ae38a","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/341//#testresult/547aba3eed5ae38a","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1730085088950,"stop":1730085088963,"duration":13}},{"uid":"9e49f1c770e7b3cb","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/337//#testresult/9e49f1c770e7b3cb","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729998792219,"stop":1729998792231,"duration":12}},{"uid":"8a9be46eef58260a","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/340//#testresult/8a9be46eef58260a","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729998660895,"stop":1729998660908,"duration":13}},{"uid":"729aafdb5a4af714","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/336//#testresult/729aafdb5a4af714","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729912099104,"stop":1729912099117,"duration":13}},{"uid":"a444a78e2f9d44ee","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/339//#testresult/a444a78e2f9d44ee","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729911958648,"stop":1729911958661,"duration":13}},{"uid":"5fa1fed6d26b2275","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/335//#testresult/5fa1fed6d26b2275","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729825909895,"stop":1729825909908,"duration":13}},{"uid":"7f0c3c23c0a7576e","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/338//#testresult/7f0c3c23c0a7576e","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729825772518,"stop":1729825772531,"duration":13}},{"uid":"f7ff22f16895a7f5","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/334//#testresult/f7ff22f16895a7f5","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729739407298,"stop":1729739407311,"duration":13}},{"uid":"d2237f638a12d855","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/337//#testresult/d2237f638a12d855","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729739259419,"stop":1729739259431,"duration":12}},{"uid":"c28c4a9efb61eb9f","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/333//#testresult/c28c4a9efb61eb9f","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729653016634,"stop":1729653016647,"duration":13}},{"uid":"d772155d3cc9c0c2","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/336//#testresult/d772155d3cc9c0c2","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729652874092,"stop":1729652874106,"duration":14}},{"uid":"4f5880e2f3590639","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/332//#testresult/4f5880e2f3590639","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729566668351,"stop":1729566668364,"duration":13}},{"uid":"c4e64ec9d8b1285a","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/335//#testresult/c4e64ec9d8b1285a","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729566532324,"stop":1729566532337,"duration":13}}]},"tags":[]},"source":"a9b9b848005a3ae.json","parameterValues":[]}